gui/src/containers/About.js

```diff
@@ -37,4 +40,13 @@ const About = ({classes, version, sha}) => {
   )
 }
 
-export default withRouteData(withStyles(styles)(About))
+const mapStateToProps = state => ({})
+
+const mapDispatchToProps = (dispatch, ownProps) => {
+  dispatch(matchRoute(ownProps.match))
```

Comment:
I see that this clears `messages` and updates the `currentUrl`. Is this how we're removing the flash? Seems less than DRY to have to sprinkle this in every page. Let's chat about it offline and make sure I understand it 😅  .

Reply:
@dimroc this is definitely something I dislike :) I wasn't able to get https://github.com/supasate/connected-react-router to work as there is no hook point for the wrapping router provided by `react-static`.

Reply:
Well, rather than sprinkle `dispatch(matchRoute(ownProps.match))` everywhere, could we just make our own version of `bindActionCreators`, say `resetRouteBindActionCreators`, that includes the call and use that helper instead?